frontend/packages/dev-console/src/components/edit-application/__tests__/edit-application-data.ts

```diff
@@ -663,4 +886,571 @@ export const knExternalImageValues: DeployImageFormData = {
+          resourceVersion: '676247',
+          name: 'nationalparks-py',
+          uid: '2bc985ac-f834-45e5-9a86-830edb6bc8bd',
+          creationTimestamp: '2020-01-15T15:51:47Z',
+          generation: 1,
+          namespace: 'div',
+          labels: {
+            app: 'nationalparks-py',
+            'app.kubernetes.io/component': 'nationalparks-py',
+            'app.kubernetes.io/instance': 'nationalparks-py',
+            'app.kubernetes.io/name': 'python',
+            'app.kubernetes.io/part-of': 'nodejs-rest-http-app',
+            'app.openshift.io/runtime': 'python',
+            'app.openshift.io/runtime-version': '3.6',
+          },
+        },
+        spec: {
+          lookupPolicy: {
+            local: false,
+          },
+        },
+        status: {
+          dockerImageRepository:
+            'image-registry.openshift-image-registry.svc:5000/div/nationalparks-py',
+        },
+      },
+    ],
+  },
+};
+
+export const gitImportInitialValuesWithHealthChecksEnabled: GitImportFormData = {
+  formType: 'edit',
+  name: 'nationalparks-py',
+  application: { name: '', selectedKey: UNASSIGNED_KEY },
+  project: { name: 'div' },
+  route: {
+    disable: true,
+    create: true,
+    targetPort: '8080-tcp',
+    unknownTargetPort: '',
+    defaultUnknownPort: 8080,
+    path: '',
+    hostname: 'nationalparks-py-div.apps.rorai-cluster34.devcluster.openshift.com',
+    secure: false,
+    tls: {
+      termination: '',
+      insecureEdgeTerminationPolicy: '',
+      caCertificate: '',
+      certificate: '',
+      destinationCACertificate: '',
+      privateKey: '',
+    },
+  },
+  resources: Resources.OpenShift,
+  serverless: {
+    scaling: {
+      minpods: 0,
+      maxpods: '',
+      concurrencytarget: '',
+      concurrencylimit: '',
+    },
+  },
+  pipeline: { enabled: false },
+  deployment: { env: [], triggers: { image: true, config: true }, replicas: 1 },
+  labels: {},
+  limits: {
+    cpu: {
+      request: '',
+      requestUnit: '',
+      defaultRequestUnit: '',
+      limit: '',
+      limitUnit: '',
+      defaultLimitUnit: '',
+    },
+    memory: {
+      request: '',
+      requestUnit: 'Mi',
+      defaultRequestUnit: 'Mi',
+      limit: '',
+      limitUnit: 'Mi',
+      defaultLimitUnit: 'Mi',
+    },
+  },
+  git: {
+    url: 'https://github.com/divyanshiGupta/nationalparks-py',
+    type: 'Git',
+    ref: '',
+    dir: '/',
+    showGitType: false,
+    secret: '',
+    urlValidation: ValidatedOptions.default,
+    isUrlValidating: false,
+  },
+  docker: { dockerfilePath: 'Dockerfile', containerPort: 8080 },
+  image: {
+    selected: 'python',
+    recommended: '',
+    tag: '3.6',
+    tagObj: {},
+    ports: [],
+    isRecommending: false,
+    couldNotRecommend: false,
+  },
+  build: {
+    env: [],
+    triggers: { webhook: true, image: true, config: true },
+    strategy: 'Source',
+  },
+  healthChecks: {
```

Comment:
Just reuse the original initial Values variable and add this property in it.

Reply:
I agree with @rohitkrai03 on this. Please try to avoid adding any duplicate code.

Reply:
For Readiness and LivenessProbes I  have different objects & for startup Probe I am using the default health check values

Reply:
I can see only a few properties change in both the objects but anyways. I guess its fine.